tests/Integration/TestCase.php

```diff
@@ -106,9 +110,10 @@ public function create_test_blog( $domain, $user_id ) {
 	 *
 	 * @param string $taxonomy_key Taxonomy key, must not exceed 32 characters.
 	 * @param string $term_name    The term name to add.
+	 *
```

Comment:
Technically the WPCS standard has no clear line between the `@param` and the `@return` lines.

Reply:
This is PHPStorm doing its things. I will revert.